src/pages/home/report/ReportActionsView.js

```diff
@@ -403,6 +407,11 @@ class ReportActionsView extends React.Component {
             return null;
         }
 
+        // Waiting until ONYX variables are loaded before displaying the component
```

Comment:
I retested this and I didn't find any case where `_.isEmpty(this.props.personalDetails)` is `true`. At this point I'm not sure why I added it.

I just removed it, I don't think it makes sense because:
- if `this.props.personalDetails` is an empty object, we won't crash when we open the DetailsPage because of the [new spinner](https://github.com/Expensify/App/blob/7b1660cb7361894d565752ad78b283cf8e3ffa9c/src/pages/DetailsPage.js#L70-L77)
- ReportActionsView doesn't really use the personal details, so it is wrong to hold it from rendering because PersonalDetails are not ready
- if `this.props.personalDetails` is `null` or `undefined`, it would crash earlier [here](https://github.com/Expensify/App/blob/7b1660cb7361894d565752ad78b283cf8e3ffa9c/src/pages/home/report/ReportActionsView.js#L134)